docs/config.rst

```diff
@@ -0,0 +1,115 @@
+.. _config:
+
+ZPUI configuration files
+========================
+
+ZPUI ``config.json``
+++++++++++++++++++++++++
+
+.. important::
+
+  By default, ZeroPhone SD card images and ZPUI installs ship with config.json files
+  that are suitable for usage out-of-the-box. Unless you want to tweak your IO drivers'
+  initialization parameters or need to debug ZPUI in case of hardware trouble,
+  you won't need to edit ZPUI configuration files.
+
+ZPUI depends on a ``config.json`` file to initialize the input and output devices. 
+To be exact, it expects a JSON-formatted file in one of the following paths (sorted by order
+in which ZPUI attempts to load them):
+
+* ``/boot/zpui_config.json``
+* ``/boot/pylci_config.json``
+* ``{ZPUI directory}/config.json``
+* ``{ZPUI directory}/config.example.json`` (a fallback file that you shouldn't edit manually)
+
+.. note::
+
+  The ``config.json`` tells ZPUI which output and input hardware it needs to use, so
+  invalid configuration might lock you out of the system. Thus, it's better to make changes
+  in ``/boot/zpui_config.json`` - if you screw up and lock yourself out of ZPUI,
```

Comment:
It is a very good idea and I didn't understand the reason for `/boot/zpui_config.json` until I read this doc. 
+1 internet for you 👍